PupDates-CrossPol/pupdates_frontend
# Firebase setup

#### What's this PR do?
Adds functionality for uploading images to firebase storage from both the camera and photo library on ios mobile
#### Where should the reviewer start?
On the selectImg method inside of LoginForm.js
#### How should this be manually tested?
Run ```npm install``` and after selecting an image, check [here](https://console.firebase.google.com/project/pupdates-b3204/storage/pupdates-b3204.appspot.com/files~2F) to see that the image has been added to the images folder inside of storage
#### What are the relevant tickets?
Issue #9 
Issue #3
